manifests/cmp-sidecar/argocd-repo-server.yaml

```
@@ -25,26 +26,12 @@ spec:
         env:
           - name: AVP_VERSION
             value: 1.11.0
-          - name: HELM_VERSION
-            value: 3.9.0
-          - name: KUSTOMIZE_VERSION
-            value: 4.5.5
         command: [sh, -c]
         args:
           - >-
             curl -L https://github.com/argoproj-labs/argocd-vault-plugin/releases/download/v$(AVP_VERSION)/argocd-vault-plugin_$(AVP_VERSION)_linux_amd64 -o argocd-vault-plugin &&
             chmod +x argocd-vault-plugin &&
-            mv argocd-vault-plugin /custom-tools/ &&
-
-            curl -L https://get.helm.sh/helm-v$(HELM_VERSION)-linux-amd64.tar.gz -o helm.tar.gz &&
-            tar -xvf helm.tar.gz &&
-            chmod +x linux-amd64/helm &&
-            mv linux-amd64/helm /custom-tools &&
-
-            curl -L https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv$(KUSTOMIZE_VERSION)/kustomize_v$(KUSTOMIZE_VERSION)_linux_amd64.tar.gz -o kustomize.tar.gz &&
-            tar -xvf kustomize.tar.gz &&
-            chmod +x kustomize &&
-            mv kustomize /custom-tools
+            mv argocd-vault-plugin /custom-tools/
```

FWIW, I realized yesterday (and didn't get to adding it to the PR) that this code could instead be 

```suggestion
            curl -L https://github.com/argoproj-labs/argocd-vault-plugin/releases/download/v$(AVP_VERSION)/argocd-vault-plugin_$(AVP_VERSION)_linux_amd64 -o /custom-tools/argocd-vault-plugin &&
            chmod +x /custom-tools/argocd-vault-plugin
```

YMMV, but the benefit here would be that the initContainer could be more secure and use 
```yaml
securityContext:
  readOnlyRootFilesystem: true
```
as well as not leaving the binary lying around on the host